zipline/data/minute_bars.py

```diff
@@ -465,7 +471,7 @@ def write(self, sid, df):
             'volume': df.volume.values,
         }
         dts = df.index.values
-        self.write_cols(sid, dts, cols)
+        self._write_cols(sid, dts, cols)
```

Comment:
Could you add a comment to explain why we don't need to call `write_cols` here, basically what you said in the PR description?
> Make a separate _write_cols method for use by both write_cols and write, since the write method which takes a DataFrame has the matched input length enforced by the DataFrame.